### DISCUSSION TITLE
Welcome to TestimonialFree Discussions!

### DISCUSSION THREAD

**Discussion by vinodsharma10x** [careerJumpstart/TestimonialFree#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nitesh-chowdhary** [comment]
Hi, I'm Nitesh, currently learning frontend development and willing to contribute in open source

**Reply by ClaraLopezUribe** [reply]
Hello Nitesh!

**Comment by ClaraLopezUribe** [comment]
Hello All!

I am Clara.  I am currently learning front end development (HTML, CSS & Bootstrap, and vanilla Javascript). In the next few weeks, I will continue my studies in the .Net framework (C#, ASP.NET MVC, PostgreSQL/pgAdmin).

**Comment by Saimurugeshwari** [comment]
Hello All,
I am Murugeshwari, I am learning front end development, and this is my first ever contribution in an open-source project, In a next few week I will continue to study about frameworks. (react and nextjs).
